mw-webapp/src/service/auth/constants.ts

```diff
@@ -0,0 +1,4 @@
+export const USER_IS_AUTH = "userIsAuth";
+export const PATH = "path";
+export const USER_REGISTERED = "true";
```

Comment:
Strange naming

Reply:
lets get rid of this file. ;Constants' mean nothing, i could put in constant anything, also this style lead to situation when you have 100500 files with abstract name like 'constants`, 'helperts' and navigation by file name in this projects is a nightmare :)

Reply:
rename file